server/client.go

```diff
@@ -932,6 +1045,11 @@ func (c *client) processMsg(msg []byte) {
 	}
 }
 
+func (c *client) pubPermissionViolation(subject []byte) {
+	c.sendErr(fmt.Sprintf("Permissions Violation for Publish to %q", subject))
+	c.Errorf("Publish Violation - User %q, Subject %q", c.opts.Username, subject)
```

Comment:
I had it as a debug statement.. Not sure. I need to start looking at the clients and make sure they easily do the right thing so we can avoid, my app doesn't work, etc..